tools/build_defs/pkg/archive.py

```diff
@@ -20,6 +20,8 @@
 import subprocess
 import tarfile
 
+# Use a deterministic mtime that doesn't confuse other programs.
```

Comment:
Can you refer to the bug here or change to 
"other programs such as python distutils".

Reply:
I'll make the change while merging.

Reply:
Sorry for the delay. Had a few emergencies and lots of distractions yesterday. Just pushed an update; hope the new text suffices.